dll/win32/shell32/CShellLink.cpp

```diff
@@ -1721,6 +1721,9 @@ static HRESULT SHELL_PidlGetIconLocationW(PCIDLIST_ABSOLUTE pidl,
 HRESULT STDMETHODCALLTYPE CShellLink::GetIconLocation(UINT uFlags, PWSTR pszIconFile, UINT cchMax, int *piIndex, UINT *pwFlags)
 {
     HRESULT hr;
+
+    pszIconFile[0] = 0;
```

Comment:
`L'\0'` value is more explicit.
Maybe use `*pszIconFile`, here and below?

Reply:
`pszIconFile[0]` is just fine.

Reply:
OK.